tipuesearch_lite.js

```diff
@@ -152,34 +140,30 @@ window.onload = function execute(){
     }
 
     function selectPageText(pageText, searchTerms) {
-        let tempPageText = pageText.toLowerCase();
-        let posSearchTerm = -1;
-        for (const term of searchTerms) {
-            posSearchTerm = tempPageText.indexOf(term);
-            if (posSearchTerm != -1) {
-                break;
+        let tempText = pageText;
+        let bestText = "";
+        let score = 0;
```

Comment:
This is only used inside of the `while` loop, so it can be removed from here, and the scope can be tightened to inside the loop.